site/en/blog/new-in-devtools-114/index.md

```diff
@@ -139,6 +139,21 @@ Additionally, the ID of the interaction has been moved from the tooltip to **Sum
 
 Chromium issues: [1432512](https://crbug.com/1432512), [1432509](https://crbug.com/1432509).
 
+### Web Vitals moved to Timings {: #web-vitals }
+
+The **Performance** panel gets rid of the following redundancies:
+
+- Non-interactable **Web Vitals** track. Instead, see the relevant timings in the **Timings** track on hover.
+- The **Long Tasks** subtrack. You can already find such tasks in the [**Main** track](/docs/devtools/performance/reference/#long-tasks) shaded [in red and with a red triangle](/blog/new-in-devtools-83/#long-tasks).
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/Wu4ePOcdvtKHyvy0PK96.png", alt="Before and after moving the Web Vitals to the Timings track.", width="800", height="613" %}
+
+Learn more about [Web Vitals](https://web.dev/vitals/).
+
+{# https://chromium-review.googlesource.com/q/Ib6f7a8788298187a70bdb44820ce452c6d8db1f1 #}
+
+Chromium issue: [1430844](https://crbug.com/1430844).
+
```

Comment:
Personally I still think it's worth noting the Layout Shifts change. We can always update it if/when it changes again. But let's keep it short:

```suggestion
Chromium issue: [1430844](https://crbug.com/1430844).

The Experiences track was renamed to Layout Shifts to more accurately reflect it's usage.

{# https://chromium-review.googlesource.com/q/26306da9622791b255b12c9c0deeac0a0d4a07b8 #}

Chromium issue: [4403716](https://crbug.com/1428522).
```

Do we want to include a screenshot here?

Reply:
I think we're fine. It's just a renaming.